addons/web/static/src/xml/graph.xml

```diff
@@ -12,6 +12,7 @@
                 <t t-if="props.isSample and !props.isEmbedded and props.noContentHelp" t-call="web.ActionHelper">
                     <t t-set="noContentHelp" t-value="props.noContentHelp"/>
                 </t>
+                <t t-elif="props.isSample and !props.isEmbedded" t-call="web.NoContentHelper"/>
```

Comment:
I think @Polymorphe57 had a similar issue with the dashboard, i'm not sure though

Reply:
The only reason to have a NoContentHelper is when a mix of positive and negative measure values is obtained in the "pie" mode. A chart possibly empty had to be displayed in all other situations. Are we sure we want to change that? If yes, I would write the conditions in another way. First a t-if="props.isSample and !props.isEmbedded" then a t-if="props.noContentHelp" and t-else=""

Reply:
I have done changes. You can check it out